### PR TITLE
My py fixes

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
Ready

## Description
update build to work with a specific mypy version instead of latest.
made some recommended changes to integration from the nightly build (with the newer version of mypy)

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: dev-requirements-py3.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
 flake8>=3.7.8
-demisto-sdk>=0.3.3
-demisto-py>=2.0.7
+demisto-sdk==0.3.3
+demisto-py==2.0.8
```
Comment (liorblob) [inline dev-requirements-py3.txt]: not sure about these... @ronykoz what do you think?

Comment (yaakovi) [reply dev-requirements-py3.txt]: 2.0.8 introduced updates regarding ssl certificate, it's not mandatory to increase the version but shouldn't cause any issues.

Comment (liorblob) [reply dev-requirements-py3.txt]: We might want the SDK to be always latest

Comment (yaakovi) [reply dev-requirements-py3.txt]: we want the option to release a new SDK version but keep working on a stable version for a few more days. (wouldn't like to release a new sdk in middle of content release and break the build).

Comment (liorblob) [reply dev-requirements-py3.txt]: every SDK version is stable!

**Pull request merged**